src/features/placeDetail/PhotoGrid.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+const GridContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  max-width: 50rem;
+  max-height: 40rem;
+  overflow-y: auto;
+  padding: 0.1rem;
+  gap: 0.1rem;
+  background: var(--background-color);
+`;
+
+const GridImage = styled.img`
+  width: 100%;
+  aspect-ratio: 1 / 1;
+  object-fit: cover;
+  cursor: pointer;
+`;
+
+interface PhotoGridProps {
+  images: string[];
+  onImageClick: (index: number) => void;
+}
+
+const PhotoGrid: React.FC<PhotoGridProps> = ({ images, onImageClick }) => {
```

Comment:
컴포넌트 선언할 때 `React.FC`는 되도록 지양하시는게 좋아요!
```suggestion
const PhotoGrid = ({ images, onImageClick }: PhotoGridProps) => {
```
이렇게 하시는건 어떠세요?